Makefile

```diff
@@ -92,6 +92,11 @@ build-linux: go.sum
 	env GOOS=linux GOARCH=amd64 go build -mod=readonly -o ./build/Linux-AMD64/cncli $(BUILD_FLAGS) ./cmd/cncli
 	env GOOS=linux GOARCH=amd64 go build -mod=readonly -o ./build/Linux-AMD64/cnd $(BUILD_FLAGS) ./cmd/cnd
 
+build-local-linux: go.sum
```

Comment:
Why is this target needed? Can't we mount `build/Linux-AMD64` as `/cnd` in docker?

Reply:
I think that the original idea was to compile for linux the binaries for the base docker: see "LEDGER_ENABLED=false GOOS=linux GOARCH=amd64 $(MAKE) build". 
I just wanted to keep the same procedure.

Reply:
Why can't we use `build-linux` as a dependency in `localnet-start`?

Reply:
Because if we had made this change we would have had to also change build-docker-cndode to point to the correct compiled and make more changes to docker-compose.yaml. I chose the method that involved fewer changes